src/javascript/crypto/e2e/vrf/extra25519_test.html

```diff
@@ -0,0 +1,56 @@
+<!-- Copyright 2015 Yahoo Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -->
+<!DOCTYPE html>
+<html>
+<head>
+<title>Unit Test of e2e.vrf.extra25519</title>
+<script src="../../../../../javascript/closure/base.js"></script>
+<script src="test_js_deps-runfiles.js"></script>
+<script>
+goog.require('goog.testing.jsunit');
+
+goog.require('e2e.vrf.sha3Shake256');
+goog.require('e2e.vrf.extra25519');
+
+goog.require('goog.crypt');
+
+</script>
+<script>
+
+
+/**
+ * Tests that e2e.vrf.extra25519.fromBytesBaseGroup works, and no assertion error is thrown
+ */
+function testFromBytesBaseGroup() {
+	var pk = [194, 191, 96, 139, 106, 249, 24, 253, 198, 131, 88, 169, 100, 231, 7, 211, 70, 171, 171, 207, 24, 30, 150, 114, 77, 124, 240, 123, 191, 14, 29, 111];
+
+	e2e.vrf.extra25519.fromBytesBaseGroup(pk);
```

Comment:
gd catch. assert statements were once inlined in `fromBytesBaseGroup()`, but later removed. will update it accordingly